### PR TITLE
Telebatons for heads of staff

### PR DESCRIPTION
**Pull request by ghost**
3 tick knockdown (not stun) on help/disarm/grab intent
10 force (no knockdown as well) on harm intent (so no chainstun beating)

Does not replace flashes for now, might remove flashes from the head lockers/offices in the future but right now it's fine.

This also does a few more things:
- Changes woodhit.ogg a bit
- Replaces chain of command and classic baton stun noise with woodhit
- Fixes 'you push up your the gas mask' and stuff

### CODE REVIEW

File: code/game/objects/items/weapons/melee/misc.dm [hunk 1]
```diff
@@ -72,3 +72,79 @@
...
+	item_state = "telebaton_0"
+	slot_flags = SLOT_BELT
+	w_class = 2
+	force = 3
+	var/on = 0
+
+/obj/item/weapon/melee/telebaton/attack_self(mob/user as mob)
+	on = !on
+	if(on)
+		user.visible_message("\red With a flick of their wrist, [user] extends their telescopic baton.",\
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/melee/misc.dm]: > /red

File: code/game/objects/items/weapons/melee/misc.dm [hunk 2]
```diff
@@ -72,3 +72,79 @@
...
+		"\red You extend the baton.",\
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/melee/misc.dm]: > /red

File: code/game/objects/items/weapons/melee/misc.dm [hunk 3]
```diff
@@ -72,3 +72,79 @@
...
+		"You hear an ominous click.")
+		icon_state = "telebaton_1"
+		item_state = "nullrod"
+		w_class = 3
+		force = 10//seclite damage
+		attack_verb = list("smacked", "struck", "cracked", "beaten")
+	else
+		user.visible_message("\blue [user] collapses their telescopic baton.",\
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/melee/misc.dm]: > /blue

File: code/game/objects/items/weapons/melee/misc.dm [hunk 4]
```diff
@@ -72,3 +72,79 @@
...
+		"\blue You collapse the baton.",\
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/melee/misc.dm]: > /blue

File: code/game/objects/items/weapons/melee/misc.dm [hunk 5]
```diff
@@ -72,3 +72,79 @@
...
+		blood_overlay = I
+
+		overlays += blood_overlay
+
+	return
+
+/obj/item/weapon/melee/telebaton/attack(mob/target as mob, mob/living/user as mob)
+	if(on)
+		if ((CLUMSY in user.mutations) && prob(50))
+			user << "\red You club yourself over the head."
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/melee/misc.dm]: > /red

File: code/game/objects/items/weapons/melee/misc.dm [hunk 6]
```diff
@@ -72,3 +72,79 @@
...
+			return
+		if (user.a_intent == "harm")
+			if(!..()) return
+			if(!isrobot(target))
+				playsound(get_turf(src), "swing_hit", 50, 1, -1)
+		else
+			playsound(get_turf(src), 'sound/effects/woodhit.ogg', 75, 1, -1)
+			target.Weaken(3)
+			src.add_fingerprint(user)
+			target.visible_message("\red <B>[target] has been knocked down with \the [src] by [user]!</B>")
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/melee/misc.dm]: > /red

**Comment by ghost** [comment]
oh jeez i forgot how outdated that shit is, one second.

**Comment by Iamgoofball** [comment]
also ignore the travis failed for now, it's because of honk

**Comment by RemieRichards** [comment]
Merging, I'll fix the Remie/Goof incident.

**Pull request merged**